Lib/Mpl/Types.hpp

```diff
@@ -12,7 +12,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ****************************************************************************/
 #pragma once
-#define BRIGAND_NO_BOOST_SUPPORT 1
```

Comment:
I'm not sure if the current brigand needs this but we probably should leave it in. MPL will be refactored in the future as well. Right now it works but it could be written in a more "best practices" manor. I think its important that other people be able to read my TMP guts and I'm not sure that is the case right now.